### ISSUE TITLE
Error while running initial backup

### ISSUE THREAD

**Issue by borpin** [joeyates/imap-backup#85]
Saw this error during an initial backup.  Rerunning started from where it left off.

Running under Ubuntu 20.04 as a Proxmox LXC.

```
D, [2020-10-22T07:54:56.843152 #312] DEBUG -- : [INBOX] uid: 61446 (12214/20208) - 43660 bytes
Traceback (most recent call last):
        25: from /usr/local/bin/imap-backup:23:in `<main>'
        24: from /usr/local/bin/imap-backup:23:in `load'
        23: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/bin/imap-backup:73:in `<top (required)>'
        22: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/configuration/list.rb:18:in `each_connection'
        21: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/configuration/list.rb:18:in `each'
        20: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/configuration/list.rb:20:in `block in each_connection'
        19: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/account/connection.rb:49:in `run_backup'
        18: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/account/connection.rb:90:in `each_folder'
        17: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/account/connection.rb:90:in `each'
        16: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/account/connection.rb:93:in `block in each_folder'
        15: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/account/connection.rb:54:in `block in run_backup'
        14: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/downloader.rb:15:in `run'
        13: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/downloader.rb:15:in `with_index'
        12: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/downloader.rb:15:in `each'
        11: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/downloader.rb:16:in `block in run'
        10: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/account/folder.rb:68:in `fetch'
         9: from /var/lib/gems/2.7.0/gems/imap-backup-2.2.2/lib/imap/backup/account/folder.rb:91:in `examine'
         8: from /usr/lib/ruby/2.7.0/net/imap.rb:470:in `examine'
         7: from /usr/lib/ruby/2.7.0/monitor.rb:202:in `mon_synchronize'
         6: from /usr/lib/ruby/2.7.0/monitor.rb:202:in `synchronize'
         5: from /usr/lib/ruby/2.7.0/net/imap.rb:472:in `block in examine'
         4: from /usr/lib/ruby/2.7.0/net/imap.rb:1255:in `send_command'
         3: from /usr/lib/ruby/2.7.0/monitor.rb:202:in `mon_synchronize'
         2: from /usr/lib/ruby/2.7.0/monitor.rb:202:in `synchronize'
         1: from /usr/lib/ruby/2.7.0/net/imap.rb:1273:in `block in send_command'
/usr/lib/ruby/2.7.0/net/imap.rb:1213:in `get_tagged_response': end of file reached (EOFError)
```

**Comment by joeyates** [comment]
Hi @borpin,

It looks like this is a network-induced error. I agree that the error message `EOFError` is not very helpful, but as it's happening in Net::IMAP, there's not much we can do.

**Comment by joeyates** [comment]
@borpin This is fixed in version 3.2.0
